src/transformers/models/opt/modeling_tf_opt.py

```diff
@@ -53,6 +53,9 @@
 # Base model docstring
 _EXPECTED_OUTPUT_SHAPE = [1, 8, 1024]
 
+# Causal LM output
+_EXPECTED_CAUSAL_LM_OUTPUT = "Hey, are you consciours? Can you talk to me?\nI'm not consciours, but I can talk to you."
```

Comment:
Maybe `_CAUSAL_LM_EXPECTED_OUTPUT` to keep aligned with other names?

Reply:
Good catch